### PR TITLE
Create list-of-all-elements

### PR DESCRIPTION
**Pull request by amy-silcock**
Need to check
Would like Recommended (minimum) to be over two lines
Like other table, would be good if we could shade every other Topic.

### CODE REVIEW

**Comment by hayfield** [comment]
> Would like Recommended (minimum) to be over two lines

Because of the length of the heading, it's likely that this will occur during the calculation of default widths.

> Like other table, would be good if we could shade every other Topic.

Standard table styling will be applied, so nothing needs to be done here.

File: en/what-do-i-need-to-publish/list-of-all-elements [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Full list of top-level elements in IATI
+=========
+
```
Comment (hayfield) [inline en/what-do-i-need-to-publish/list-of-all-elements]: Suggest adding some sort of introductory text to explain what the table is about, and what the columns mean.

File: en/what-do-i-need-to-publish/list-of-all-elements [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| Basic Activity Information | Title | Yes |
+|  | Description | Yes |
+|  | Activity Status | Yes | 
+|  | Activity Date | Yes |
+|  | Contact Info |
+|  | Activity Scope |
+| Participating Organisations | Participating Org | Yes |
+| Geopolitical Information | Recipient Country |  | Yes |
+|  | Or, Recipient Region |  | Yes |
+|  | Location |
```
Comment (hayfield) [inline en/what-do-i-need-to-publish/list-of-all-elements]: There are a number of rows with nothing in either the `Mandatory` or `Recommended (minimum)` column. This seems a bit confusing.

File: en/what-do-i-need-to-publish/list-of-all-elements [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Full list of top-level elements in IATI
```
Comment (hayfield) [inline en/what-do-i-need-to-publish/list-of-all-elements]: The file should have a `.rst` extension.

File: en/what-do-i-need-to-publish/list-of-all-elements (hunk 2 above)
Comment (amy-silcock) [reply en/what-do-i-need-to-publish/list-of-all-elements]: It follows from the previous page 'what do I need to publish'. But this time with blank rows of all the non 'minimum' elements.

**Comment by hayfield** [comment]
@amy-silcock The file is still missing the `.rst` filetype extension.

**Comment by dalepotter** [comment]
Suggest that this PR is closed, as these changes will now appear in the IATI-Extra-Documentation repo, since it contains version specific content. This was agreed following discussion with @amy-silcock and @PetyaKangalova

**Pull request closed without merging**